src/content/docs/2/guide/recipe/notifications.mdx

```diff
@@ -0,0 +1,112 @@
+description: Stay tuned while we work on some delicious recipes.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+In this recipe we'll take a look at sending a native notification to your user. We'll do this by using the official notification plugin.
+
+## Setup
+
+First you have to install the notification plugin in your backend.
+
+<Tabs>
+    <TabItem label="Auto">
+    ```bash
+    tauri plugin add notification
+    ```
+    </TabItem>
+    <TabItem label="Manual">
+        <Tabs>
+            <TabItem label="Cargo.toml">
+            ```toml
+            [dependencies]
+            tauri-plugin-notification = "2.0.0-alpha"
+            ```
+            </TabItem>
+            <TabItem label="lib.rs">
+            ```rust
+            #[cfg_attr(mobile, tauri::mobile_entry_point)]
+            pub fn run() {
+                tauri::Builder::default()
+                    // Initialize the plugin
+                    .plugin(tauri_plugin_notification::init())
+                    .run(tauri::generate_context!())
+                    .expect("error while running tauri application");
+            }
+            ```
+            </TabItem>
+        </Tabs>
+    </TabItem>
+</Tabs>
+
+## Usage
+
+Once you're done with the general setup you're free to use the notification plugin in either your frontend or straight from the backend.
+
+### Frontend
+
+Install the `@tauri-apps/plugin-notification` package using your favorite package manager.
+
+```js
+import {
+	isPermissionGranted,
+	requestPermission,
+	sendNotification,
+} from '@tauri-apps/plugin-notification';
+
+// Do you have permission to send a notification?
+let permissionGranted = await isPermissionGranted();
+
+// If not we need to request it
+if (!permissionGranted) {
+	const permission = await requestPermission();
+	permissionGranted = permission === 'granted';
+}
+
+// Once permission has been granted we can send the notification
+if (permissionGranted) {
+	sendNotification({ title: 'TAURI', body: 'Tauri is awesome!' });
+}
+```
+
+### Backend
+
+You can also create notifications straight from the backend. All you need to do is create an instance of `Notification` and `.show()` it.
+
+<Tabs>
+    <TabItem label="Command">
+    ```rust
+    use tauri_plugin_notification::Notification;
+
+    #[tauri::command]
+    async fn notify(app: AppHandle) -> Result<(), ()> {
+        Notification::new(&app.config().tauri.bundle.identifier)
+            .title("New message")
+            .body("You've got a new message.")
+            .show()
+    }
+    ```
+    </TabItem>
+    <TabItem label="Window event">
+    ```rust
+    ```
+    </TabItem>
+    <TabItem label="Setup function">
+    ```rust
+    ```
+    </TabItem>
+
+</Tabs>
+
+## Deep dive
+
+### How it works
+
+The API sends a signal over the IPC to the backend. After that either the `notify-rust` or `win7-notifications` crate
+takes care of sending a notification to the users system.
+
+### Security
```

Comment:
I really love this type of section. I think this is a really good place we could mark a `TODO:` to write a concept on sanitisation of user input and then link to it from here.

Reply:
Yea I feel like as a project with a strong sense for security this sort of section would be really appreciated by people.